playbooks/roles/simpleca/tasks/main.yaml

```diff
@@ -38,7 +38,11 @@
     path: "{{ cert_dir }}/{{ cert_user }}.csr"
     privatekey_path: "{{ user_key.filename }}"
     common_name: "{{ cert_name }}"
-    subject_alt_name: "DNS:{{ cert_name }}"
+    subject_alt_name:
+      - "DNS:{{ cert_name }}"
+      - "DNS:{{ standalone_host }}"
+      - "IP:{{ public_api }}"
+      - "IP:{{ control_plane_ip }}"
```

Comment:
in reality I think we only need this one ^
but added the other ones in case we need to re-use the certificate. It's for dev anyway?